sdk-python/tests/test_workflow.py

```diff
@@ -795,6 +796,285 @@ def my_entrypoint(thread: ThreadBuilder) -> None:
             ),
         )
 
+    def test_handle_any_failure(self):
+        def my_interrupt_handler(thread: ThreadBuilder) -> None:
+            thread.execute("my-task")
+
+        def my_entrypoint(thread: ThreadBuilder) -> None:
+            node = thread.execute("fail")
+            thread.handle_any_failure(node, my_interrupt_handler)
+            thread.execute("my-task")
+
+        wf = Workflow("my-wf", my_entrypoint)
+        self.assertEqual(
+            wf.compile(),
+            PutWfSpecRequest(
```

Comment:
This works! You just had to out-do my tests in Go :laughing: I just verified specific node's. Well done